tests/test_routes.py

```diff
@@ -413,13 +413,13 @@ def test_update_non_existent_shopcart(self):
     def test_update_shopcart_with_invalid_request_body(self):
         """It should return a 404 Not Found response for invalid request body"""
         shopcart = ShopcartFactory()
-        resp = self.client.post(self.base_url, json=shopcart.serialize())
+        resp = self.client.post(self.base_url_restx, json=shopcart.serialize())
         self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
 
         new_shopcart = resp.get_json()
         new_shopcart["name"] = "DevOps"
         resp = self.client.put(
-            f"{self.base_url}/{new_shopcart['id']}",
+            f"{self.base_url_restx}/{new_shopcart['id']}",
             json=new_shopcart['items'],
             content_type=DEFAULT_CONTENT_TYPE
```

Comment:
could you help to refactor `def test_unsupported_method_on_shopcart()` accordingly? Thanks a lot!

Reply:
`def test_update_a_shopcart_with_invalid_content_type()` at [line:829](https://github.com/CSCI-GA-2820-SU23-001/shopcarts/pull/143/files#diff-9ff873174f61a082827c26fc578df6fea7b1f6b51abbd415660645046e140b65R829) is also a test case for the Update Shopcarts API, could you help (1) refactor it accordingly and (2) move it up to line:428 (e.g. right after `def test_update_shopcart_with_invalid_request_body()`) so that all test cases of the Update Shopcarts API can stay together? Thanks a lot!!

Reply:
done